<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>942</width>
    <height>597</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Downloader</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <activeon>icons8_downloading_updates_512px.png</activeon>
   </iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>0</y>
      <width>811</width>
      <height>551</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="documentMode">
     <bool>true</bool>
    </property>
    <property name="tabsClosable">
     <bool>false</bool>
    </property>
    <property name="movable">
     <bool>false</bool>
    </property>
    <property name="tabBarAutoHide">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="tab_6">
     <attribute name="icon">
      <iconset>
       <activeon>icons8_home_60px.png</activeon>
      </iconset>
     </attribute>
     <attribute name="title">
      <string>Home</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>40</y>
        <width>271</width>
        <height>181</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>91</width>
         <height>131</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>icons8_downloading_updates_96px.png</pixmap>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>30</y>
         <width>131</width>
         <height>131</height>
        </rect>
       </property>
       <property name="text">
        <string>Download Any Types Of Files</string>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>40</y>
        <width>271</width>
        <height>181</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>30</y>
         <width>131</width>
         <height>131</height>
        </rect>
       </property>
       <property name="text">
        <string>Youtube Video Download</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>91</width>
         <height>131</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>icons8_vegas_96px.png</pixmap>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>290</y>
        <width>271</width>
        <height>181</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>30</y>
         <width>131</width>
         <height>131</height>
        </rect>
       </property>
       <property name="text">
        <string>Youtube Playlist Download</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>91</width>
         <height>131</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>icons8_repeat_96px.png</pixmap>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>290</y>
        <width>271</width>
        <height>181</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>30</y>
         <width>131</width>
         <height>131</height>
        </rect>
       </property>
       <property name="text">
        <string>App Themes &amp; Setting</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>91</width>
         <height>131</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>icons8_adjust_96px.png</pixmap>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="icon">
      <iconset>
       <activeon>icons8_download_60px.png</activeon>
      </iconset>
     </attribute>
     <attribute name="title">
      <string>Download</string>
     </attribute>
     <widget class="QPushButton" name="borwseOther">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>100</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Old Antic Outline Shaded</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Browes</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>icons8_search_52px_2.png</normaloff>icons8_search_52px_2.png</iconset>
      </property>
     </widget>
     <widget class="QLineEdit" name="otherUrl">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>50</y>
        <width>651</width>
        <height>31</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string> Enter url here</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="otherLocation">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>110</y>
        <width>441</width>
        <height>31</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string> your saved location</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="OtherprogressBar">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>360</y>
        <width>691</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
     <widget class="QPushButton" name="downloadOther">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>420</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Old Antic Outline Shaded</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Download</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>downloads_folder_96px.png</normaloff>downloads_folder_96px.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>45</width>
        <height>40</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="errormsg">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>490</y>
        <width>321</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MT Extra</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="icon">
      <iconset>
       <activeon>icons8_youtube_60px_1.png</activeon>
      </iconset>
     </attribute>
     <attribute name="title">
      <string>youtube</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>777</width>
        <height>484</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <property name="documentMode">
       <bool>false</bool>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <property name="movable">
       <bool>false</bool>
      </property>
      <property name="tabBarAutoHide">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>video</string>
       </attribute>
       <widget class="QLineEdit" name="singlelocation">
        <property name="geometry">
         <rect>
          <x>49</x>
          <y>110</y>
          <width>511</width>
          <height>31</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string> your saved location</string>
        </property>
       </widget>
       <widget class="QProgressBar" name="singlebar">
        <property name="geometry">
         <rect>
          <x>49</x>
          <y>350</y>
          <width>691</width>
          <height>23</height>
         </rect>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
       <widget class="QLineEdit" name="singleurl">
        <property name="geometry">
         <rect>
          <x>49</x>
          <y>50</y>
          <width>641</width>
          <height>31</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string> Enter url here</string>
        </property>
       </widget>
       <widget class="QComboBox" name="singlecombo">
        <property name="geometry">
         <rect>
          <x>389</x>
          <y>220</y>
          <width>231</width>
          <height>31</height>
         </rect>
        </property>
        <property name="acceptDrops">
         <bool>false</bool>
        </property>
        <property name="frame">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="gitsingledata">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>210</y>
          <width>220</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Old Antic Bold</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>get available quality</string>
        </property>
       </widget>
       <widget class="QPushButton" name="singleb">
        <property name="geometry">
         <rect>
          <x>570</x>
          <y>100</y>
          <width>121</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Old Antic Outline Shaded</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Browes</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>icons8_search_52px_2.png</normaloff>icons8_search_52px_2.png</iconset>
        </property>
       </widget>
       <widget class="QPushButton" name="singleD">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>400</y>
          <width>151</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Old Antic Outline Shaded</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Download</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>downloads_folder_96px.png</normaloff>downloads_folder_96px.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>45</width>
          <height>40</height>
         </size>
        </property>
       </widget>
       <widget class="QLabel" name="downloaded">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>310</y>
          <width>191</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="rate">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>310</y>
          <width>171</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="time">
        <property name="geometry">
         <rect>
          <x>520</x>
          <y>310</y>
          <width>151</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>playlist</string>
       </attribute>
       <widget class="QLineEdit" name="playloc">
        <property name="geometry">
         <rect>
          <x>49</x>
          <y>110</y>
          <width>511</width>
          <height>31</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string> your saved location</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="playurl">
        <property name="geometry">
         <rect>
          <x>49</x>
          <y>50</y>
          <width>641</width>
          <height>31</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string> Enter url here</string>
        </property>
       </widget>
       <widget class="QProgressBar" name="playbar">
        <property name="geometry">
         <rect>
          <x>49</x>
          <y>350</y>
          <width>691</width>
          <height>23</height>
         </rect>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
       <widget class="QComboBox" name="playquality">
        <property name="geometry">
         <rect>
          <x>480</x>
          <y>180</y>
          <width>130</width>
          <height>31</height>
         </rect>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>231</height>
         </size>
        </property>
        <property name="acceptDrops">
         <bool>false</bool>
        </property>
        <property name="frame">
         <bool>false</bool>
        </property>
        <item>
         <property name="text">
          <string>1080</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>720</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>480</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>360</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>144</string>
         </property>
        </item>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>180</y>
          <width>191</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="text">
         <string>choose the quality</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>540</x>
          <y>260</y>
          <width>41</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Total</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>260</y>
          <width>101</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Downloaded</string>
        </property>
       </widget>
       <widget class="QLCDNumber" name="lcdNumber">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>290</y>
          <width>81</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color:rgb(85, 0, 255)</string>
        </property>
       </widget>
       <widget class="QLCDNumber" name="lcdNumber_2">
        <property name="geometry">
         <rect>
          <x>540</x>
          <y>290</y>
          <width>81</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color:rgb(81, 255, 6)</string>
        </property>
       </widget>
       <widget class="QPushButton" name="playB">
        <property name="geometry">
         <rect>
          <x>570</x>
          <y>100</y>
          <width>121</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Old Antic Outline Shaded</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Browes</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>icons8_search_52px_2.png</normaloff>icons8_search_52px_2.png</iconset>
        </property>
       </widget>
       <widget class="QPushButton" name="playD">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>400</y>
          <width>151</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Old Antic Outline Shaded</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Download</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>downloads_folder_96px.png</normaloff>downloads_folder_96px.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>45</width>
          <height>40</height>
         </size>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>215</y>
          <width>211</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>260</y>
          <width>151</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>305</y>
          <width>161</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>390</x>
          <y>260</y>
          <width>61</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Rimain</string>
        </property>
       </widget>
       <widget class="QLCDNumber" name="lcdNumber_3">
        <property name="geometry">
         <rect>
          <x>390</x>
          <y>290</y>
          <width>81</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(170, 0, 0);</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="icon">
      <iconset>
       <activeon>icons8_settings_48px.png</activeon>
      </iconset>
     </attribute>
     <attribute name="title">
      <string>Setting</string>
     </attribute>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>290</y>
        <width>161</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>Default</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>290</y>
        <width>161</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>Dark</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>290</y>
        <width>161</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>Blue</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>290</y>
        <width>161</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>Green</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="home">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>101</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icons8_home_100px.png</normaloff>icons8_home_100px.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>160</width>
      <height>158</height>
     </size>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="you">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>101</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icons8_youtube_100px.png</normaloff>icons8_youtube_100px.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>150</width>
      <height>150</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="down">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>101</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icons8_download_100px.png</normaloff>icons8_download_100px.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>137</width>
      <height>153</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="setting">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>101</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icons8_settings_100px.png</normaloff>icons8_settings_100px.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>150</width>
      <height>150</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>942</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
